locales/es.json

```diff
@@ -0,0 +1,898 @@
+	"app.filters.auction.type": "Tipo",
+    "app.filters.auction.state": "Estado",
+    "app.filters.auction.category": "Categoria",
+
+    "app.regions.en": "Internacional (EN)",
+    "app.regions.ru": "Rusia",
+    "app.regions.ko": "Korea",
+    "app.regions.fr": "Francia",
+    "app.regions.sv": "Suecia",
+    "app.regions.de": "Alemania",
+    "app.regions.pl": "Polonia",
+    "app.regions.zh": "China",
+
+    "app.mods.common": "Común",
+    "app.mods.uncommon": "Poco Común",
+    "app.mods.rare": "Raro",
+    "app.mods.legendary": "Legendario",
+    "app.mods.peculiar": "Peculiar",
+    "app.mods.riven": "Riven",
+	"app.mods.lich": "Lich",
+	
+	"app.quirk.notSet": "No establecido",
+
+    "app.sort.label": "Ordenar por",
+    "app.sort.itemName": "Nombre",
+    "app.sort.createDate": "Fecha de creación",
+    "app.sort.updateDate": "Ultima actualización",
+    "app.sort.online_status": "Estado Online",
+    "app.sort.user_reputation": "Reputación",
+    "app.sort.price": "Precio",
+    "app.sort.quantity": "Cantidad",
+    "app.sort.mod_rank": "Rango",
+
+    "app.mostRecent.title": "Warframe Market | Pedidos mas recientes de Compra y Venta {platform, select, pc {} xbox {| Xbox} ps4 {| Ps4} switch {| Switch} other {}}",
+    "app.mostRecent.header": "Pedidos mas recientes",
+    "app.mostRecent.notFound": "Pedido no encontrado",
+    "app.mostRecent.showMore": "Mostrar mas",
+    "app.mostRecent.showLess": "Mostrar menos",
+
+    "app.auctions.createAuction": "Crear subasta",
+    "app.auctions.category.title": "Categoría",
+	"app.auctions.category.all": "Todo",
+    "app.auctions.category.unveiledRiven": "Mod agrietado desvelado",
+	"app.auctions.category.kuvaLich": "Kuva lich",
+
+    "app.auctions.tips.sellOrder": "Precio de venta",
+    "app.auctions.tips.participant": "Eres participante",
+    "app.auctions.tips.own": "Tu propia subasta",
+    "app.auctions.tips.auctionType.title": "Tipo de subasta:",
+    "app.auctions.tips.auctionType.buyout.title": "Con subasta",
+    "app.auctions.tips.auctionType.buyout": " - Subastas con precio de venta.",
+    "app.auctions.tips.auctionType.sellOrder.title": "Precio de venta",
+    "app.auctions.tips.auctionType.sellOrder": " - Solo precios de venta, sin subastas, sin ofertas.{br}Si, vender pedidos en la pestaña de subastas, es un poco raro, pero flexible.",
+
+    "app.auctions.alert.reminder": "Se recomienda {reminderLink}, para estar siempre al día con sus subastas.",
+    "app.auctions.alert.reminder.link": "activar notificaciones de escritorio",
+
+    "app.auctions.tabs.mostPopular": "Popular",
+    "app.auctions.tabs.mostRecent": "Mas reciente",
+    "app.auctions.tabs.search": "Buscar",
+    "app.auctions.tabs.participatedIn": "Participando en",
+    "app.auctions.tabs.createdByMe":"Creado por mi",
+
+    "app.auctions.search.riven.title": "{itemName} Mod Agrietado | Subastas {platform, select, pc {} xbox {| Xbox } ps4 {| Ps4 } switch {| Switch } other {}}| Warframe Market",
```

Comment:
There were a few changes to the titles and descriptions in the last commit:
https://github.com/42bytes-team/wfm-localization/commit/980af6d6317d5e5e7840a0aa139469a4ae513eb7#diff-772a593ff61f0e484d43cc349a5ab31c

`Auctions ` was replaced with `Trading` along with other changes.

Can you, please, actualize this translation?

Reply:
Changed